irohad/model/converters/impl/json_query_factory.cpp

```diff
@@ -78,7 +78,7 @@ namespace iroha {
             | des.String(&Query::creator_account_id, "creator_account_id")
             | des.Uint64(&Query::query_counter, "query_counter")
             | des.Object(&Query::signature, "signature") |
```

Comment:
Broken bind operator code style

Reply:
What do you mean?

Reply:
Last bind operator should be going to next line.

Reply:
Ah, I see. Because of lambda, clang-format doesn't want to deal with it